src/coreclr/jit/llvm.cpp

```diff
@@ -42,6 +42,25 @@ struct SpilledExpressionEntry
     CorInfoType m_CorInfoType;
 };
 
+enum LocalVarLocation
```

Comment:
```suggestion
enum class LocalVarLocation
```
We prefer `enum class`es over raw `enum`s as they do not put their member into the scope they're defined in.